include/sway/input/seat.h

```diff
@@ -56,6 +56,7 @@ struct sway_seat {
 	// Operations (drag and resize)
 	enum {
 		OP_NONE,
+		OP_MOUSEDOWN,
```

Comment:
Possible to rename this to something else? This will be useful for touch and tablet too.

Reply:
Any suggestions? Cursordown maybe?

Reply:
"down" or "cursor_down" works for me